src/js/components/Box/doc.js

```diff
@@ -21,13 +20,14 @@ export default Box => schema(Box, {
       `How to align along the cross axis when contained in a Box or along
       the column axis when contained in a Grid.`,
     ],
-    // appCentered - removed
     background: [
       PropTypes.oneOfType([
         PropTypes.string,
         PropTypes.shape({
+          color: PropTypes.string,
           dark: PropTypes.bool,
           image: PropTypes.string,
+          opacity: PropTypes.string,
```

Comment:
Chris and I discussed this either being an enum with possible values: 'weak', 'medium', 'strong', or being a boolean.

Reply:
we can definitely change that to be a normalized value. Do you have suggestions on the values for each of these keys? How about?

Weak: 0.9, Medium: 0.5, Strong: 0.1 ?

Following the pattern in other components, if the user passes `true`, it would default to medium.

Reply:
My inclination is to go with the enum. Can you check with Chris though?